### ISSUE TITLE
Proposal: Wireframe and Prototype of the project

### ISSUE THREAD

**Issue by elvisron** [LinuxAccra-UserGroup/la-webapp#4]
As Lorenzo has suggested the milestone, I suggest we work on the the wireframe of our project. This will help us visualize the project. It will also inform us of what we need and need not to consider when developing the project.

I design and lo fi wireframe so that we can give feedback for redesign and improvements.

Lorenzo made few suggestions in his proposal #2 including user registration and profile. I would like us to make feature suggestions to enable me work on the wf.

**Comment by lcabrini** [comment]
Good idea. I think maybe we should utilize the wiki for this. You could upload the wireframe proposals and then we discuss them.

**Comment by elvisron** [comment]
![Uploading wireframe.png…]()

**Comment by elvisron** [comment]
Hello Guys, Been out of town for days without the internet. I have done a couple of 
work on how every page would look like. However, I would like you to review the wireframe homepage at the wiki and give feedback for improvements. Thank you.

**Comment by laudb** [comment]
Hi @elvisron, the wireframe looks good already :bow:  :+1: 
For now, I think something that shows whether a user is signed in or not because "join project" will require some sort of user sign-in/sign-up.

**Comment by laudb** [comment]
Hi @LinuxAccra-UserGroup/webapp-dev, 
 I showed the wireframe to Mr. Sabra Asante at Linux Accra today, comments are below:

 _I  agree with the informative start page, it should say this is the Linux Accra page, we are the longest lasting linux user group in Ghana and indicate that this website is being developed collaboratively on github.
  When the page is eventually set up, the landing page should be oriented to people seeking basic information: what the group is, when and where they meet, and any other interesting current issues. For example, we are thinking of participating in Software Freedom Day.
  Of course the page must give access to the members page. Again the page should be interesting by always giving new information._

 ..we will need to get him a github accnt as well :)
